olive/passes/onnx/conversion.py

```diff
@@ -27,9 +26,11 @@
 )
 from olive.model.config import IoConfig
 from olive.model.config.hf_config import HfConfig, get_model_type_from_hf_config
+from olive.model.handler.composite import CompositeModelHandler
```

Comment:
btw, is there a reason for importing this separately from the other handlers?

Reply:
Merely an oversight. Fixed.